test/OpenFeature.Contrib.Providers.FeatureManagement.Test/FeatureManagementProviderTestNoContext.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Microsoft.Extensions.Configuration;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace OpenFeature.Contrib.Providers.FeatureManagement.Test
+{
+    public class FeatureManagementProviderTestNoContext
+    {
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task MissingFlagKey_ShouldReturnDefault(bool defaultValue)
```

Comment:
for the sake of completeness, I suggest to also add test cases for the other data types (i.e. `ResolveStringValue`, etc.) to check if the default value is returned in case the flag is not found in the configuration